docs/tool/js/views/map-view.js

```diff
@@ -869,6 +871,59 @@
                 .remove();
 
         },
+        exportCsv: function() {
+          // if (data){
+            d3.select('#csvExport')
+              .on('click', function(d) {
+                  console.log("YOU CLICKED EXPORT CSV");
```

Comment:
@NealHumphrey I havent really noticed a standard message format in the console.log. I changed this to be "INFO clicked export CSV", but I can always just delete it. Let me know

Reply:
Cool, sounds good - yes not a consistent format really but we don't use the 2nd person elsewhere so that was the main thing, as well as the all caps. INFO sounds good.

Reply:
Most of the console logs that will actually stay in the final app re the ones that come from calls to setState so there's only a few other examples of things that will print like this; since this click event doesn't need to go through the state module it's an anomaly.